statistics/handle/ddl.go

```diff
@@ -55,7 +55,7 @@ func (h *Handle) HandleDDLEvent(t *util.Event) error {
 
 func getPhysicalIDs(tblInfo *model.TableInfo) []int64 {
 	pi := tblInfo.GetPartitionInfo()
-	if pi == nil {
+	if pi == nil || tblInfo.IsNewPartition {
```

Comment:
This function may be used by add index reorganize? Change it like this would cause trouble.

Reply:
this method's name is mislead - -, it's only be used in stats worker that handle ddl event.

so for a new global stats partition after ddl (e.g. add a new column) , ddl worker need handle add event and add new column info in table level(save as table id instead of partition id).